### ISSUE TITLE
Corrections in Firefox: Display of empty lines

### ISSUE THREAD

**Issue by zuphilip** [UB-Mannheim/ocr-gt-tools#44]
In Firefox, go to the corrections (`Kommentar`), then empty lines have a strange height:

![corrections-firefox](https://cloud.githubusercontent.com/assets/5199995/14891073/732926d8-0d65-11e6-9dd8-a79939b91822.png)

However, it seems that this holds for the first time you open `corrections.html`. What?! Can your replicate that?

**Comment by kba** [comment]
Not really. :question: The height issue is because of Firefox not respecting min-height for some block level elements. You mean the static page changes behavior when you reload the page?
